nixos/modules/programs/pulseview.nix

```diff
@@ -0,0 +1,28 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.pulseview;
+in
+{
+  options.programs.pulseview = {
+    enable = lib.mkEnableOption "pulseview, a sigrok GUI.";
```

Comment:
mkEnableOption adds a trailing dot, so you can remove the dot here.

Reply:
Done